src/Telegram.Bot/Types/ReplyMarkups/KeyboardButton.cs

```diff
@@ -26,8 +26,22 @@ public class KeyboardButton : IKeyboardButton
     public string Text { get; set; }
 
     /// <summary>
-    /// Optional. If <see langword="true"/>, the user's phone number will be sent as a contact when the button is pressed.
-    /// Available in private chats only
+    /// Optional. If specified, pressing the button will open a list of suitable users. Tapping on any user will send
+    /// their identifier to the bot in a “user_shared” service message. Available in private chats only.
+    /// </summary>
+    [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
+    public KeyboardButtonRequestUser? RequestUser { get; set; }
```

Comment:
should we add static helpers for new button types?

Reply:
Done